rasa/utils/train_utils.py

```diff
@@ -342,6 +368,94 @@ def override_defaults(
     return config
 
 
+def update_confidence_type(component_config: Dict[Text, Any]) -> Dict[Text, Any]:
```

Comment:
I believe this is not covered by a test. Could you please add one?

Reply:
Added a test